src/app/components/applications/SSV/MyAccount/components/Validator/ValidatorsList/ValidatorsList.tsx

```diff
@@ -145,7 +169,11 @@ const ValidatorsList = ({
       }) => validator.validator));
     } else {
       validatorsByClusterHash(1, getClusterHash(cluster.operators, walletStore.accountAddress), clusterValidatorsPagination.rowsPerPage).then((response: any) => {
-        setClusterValidators(response.validators);
+        if (response.validators && response.validators.length) {
+          setClusterValidators(response.validators);
+        } else {
+          setNoValidatorsData(true);
```

Comment:
use undefined or null as initial value for **clusterValidators** 
this way setting **[]** will have same effect as `setNoValidatorsData(true);`

Reply:
initial value for clusterValidators is empty array []

Reply:
no validator flag should trigger only if we have undefined or empty array of validators in response from API